doc/source/examples/sharp-immersed-boundary/simple-plane-model-from-composite/simple-plane-model-from-composite.rst

```diff
@@ -0,0 +1,338 @@
+
+.. warning:: 
+    The human interpretation of the composite files is cumbersome. For this reason we comments each lines.
+
+.. code-block:: text
+
+    shapes
+    0; superquadric; 0.3:2.5:0.05:2:6:2; 0.1:0:0 ; 0:-0.05:0.25 # Full left wing 
+    1; hyper rectangle; 5:5:5;0:-5:0 ; 0:0:0 # Cube to trim of the right side of the left wing
+    2; superquadric; 0.3:2.5:0.05:2:6:2; 0.1:0:0 ; 0:-0.05:-0.25 # Full right wing 
+    3; hyper rectangle; 5:5:5;0:5:0 ; 0:0:0 # Cube to trim of the left side of the right wing
+    4; superquadric; 1.5:0.3:0.3:3:2:2; -0.25:0:0 ; 0:0:0 # Fuselabe of the plane
+    5; superquadric; 0.15:0.05:0.5:2:2:6; -1.75:0:0.4 ; 0:-0.5:0 # Tail wing vertical plan
+    6; superquadric; 0.15:0.5:0.05:2:6:2; -1.85:0:0.6 ; 0:0:0 # Tail wing horizontal plan
+    7; superquadric; 0.3:0.1:0.1:5:2:2; -1.25:0.3:0.3 ; 0:0:0 # Left engine shape
+    8; cylinder; 0.05:1; -1.25:0.3:0.3 ; 0:1.57079632679:0 # Left engine hole
+    9; superquadric; 0.3:0.1:0.1:5:2:2; -1.25:-0.3:0.3 ; 0:0:0 # Right engine shape
+    10; cylinder; 0.05:1; -1.25:-0.3:0.3 ; 0:1.57079632679:0 # Right engine hole
+    11; superquadric; 0.1:0.03:0.25:6:2:2;  -1.25:0.15:0.15 ; -0.78539816:0:0 # Left engine link
+    12; superquadric; 0.1:0.03:0.25:6:2:2;  -1.25:-0.15:0.15 ; 0.78539816:0:0 # Right engine link
+    operations
+    30; difference  ; 1:0 # Trim of the left wing
+    31; difference  ; 3:2 # Trim of the right wing
+    32; union       ; 30:31 # Combined the two wings
+    33; union       ; 32:4 # Combined the wings with the plane fuselage
+    34; union       ; 33:5 # Combined vertical part of the tail wing with the fuselage
+    35; union       ; 34:6 # Combined horizontal part of the tail wing with the fuselage
+    36; difference  ; 8:7 # Hole in the left engine
+    37; union       ; 35:36 # Combined left engine with the fuselage
+    38; difference  ; 10:9 # Hole in the right engine
+    39; union       ; 37:38 # Combined right engine with the fuselage and the other engine
+    40; difference     ; 7:11 # Trim of the left engine link with the engine shape
+    41; difference     ; 9:12 # Trim of the right engine link with the engine shape
+    42; union     ; 39:40 # Combine the left engine link with the rest of the plane
+    43; union     ; 42:41 # Combine the right engine link with the rest of the plane
+  
+Lets right this file step-by-step Line-by-line:
+
+
+1. First, we create the left wing. This wing is created using a superquadric ellipsoids shape. Along the cord, we use an elongated ellipsoid with lengths 0.6 and 0.1 thickness. In the wingspane direction, the wing is defined by a 6-order power superquadric with a half wingspan of 2.5. We then tilt the wing backward by 0.25 rad and give a small angle of attack of 0.05 rad. Since the wing is now tilted backward, the right side of the superquadric shape is undesirable, so we remove it using a hypercube to trim the right side of the wing.
```

Comment:
```suggestion
1. First, we create the left wing. This wing is created using a superquadric ellipsoid shape. Along the cord, we use an elongated ellipsoid with length 0.6 and 0.1 thickness. In the wingspan direction, the wing is defined by a 6-order power superquadric with a half wingspan of 2.5. We then tilt the wing backward by 0.25 rad and give a small angle of attack of 0.05 rad. Since the wing is now tilted backward, the right side of the superquadric shape is undesirable, so we remove it using a hypercube to trim the right side of the wing.
```